lumicks/pylake/channel.py

```diff
@@ -240,9 +245,13 @@ def downsampled_over(self, range_list, reduce=np.mean, where="center"):
             cases, e.g. photon counts.
         where : str
             Where to put the final time point.
-            'center' time point is put at (timestamps_subset[0] + timestamps_subset[-1]) / 2, where timestamps_subset
-            are the timestamps corresponding to the samples being downsampled over.
-            'left' time point is put at start
+
+            - 'center'
+                The new time points are set to `(timestamps_subset[0] + timestamps_subset[-1]) / 2`,
+                where `timestamps_subset` are the timestamps corresponding to the samples being
+                downsampled over.
+            - 'left'
+                Time points are set to the starting timestamp of the downsampled data.
```

Comment:
I think if you fix up the indentation so everything matches it'll get rid of the weird formatting here and be more like the list for the color adjustment


![image](https://user-images.githubusercontent.com/61475504/187899229-0f282afd-235f-405e-878b-d42096fe3406.png)

![image](https://user-images.githubusercontent.com/61475504/187899322-0911c13d-f92c-44fe-b770-75835b246cea.png)